packages/form-js-viewer/src/render/components/util/domUtil.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Wrap CSS styles with a given prefix.
+ *
+ * @param {HTMLElement} rootNode
+ * @param {string} prefix
+ *
+ * @returns {HTMLElement}
+ */
+function wrapCSSStyles(rootNode, prefix) {
```

Comment:
I'm still not 100% sure this will for all cases, the ideal solution would be to use [`@scope`](https://developer.mozilla.org/en-US/docs/Web/CSS/@scope) but that's still experimental and node widely used.

Let's see if anyone complains

Reply:
They really keep changing the scoping mechanisms with CSS lately. 

Anyways, it's hard to guarantee, but the worst that can happen is broken CSS if people use really funky cases. If we do find these cases, we can recommend not doing whatever funky CSS stuff they are doing and fixing later.